addons/hr_expense/models/hr_expense.py

```diff
@@ -101,8 +101,42 @@ def _get_employee_id_domain(self):
 
     is_editable = fields.Boolean("Is Editable By Current User", compute='_compute_is_editable')
     is_ref_editable = fields.Boolean("Reference Is Editable By Current User", compute='_compute_is_ref_editable')
-
+    is_nonzero_product_selected =  fields.Boolean("Is product with non zero cost selected", compute='_compute_is_nonzero_product_selected')
+    is_currency_id_different = fields.Boolean("Is currency_id different from the company_currency_id", compute='_compute_is_currency_id_different')
     sample = fields.Boolean()
+    label_total_amount_company = fields.Char(compute='_compute_label_total_amount_company')
+    convert_rate_txt = fields.Char(compute='_compute_convert_rate_txt')
```

Comment:
I'd use `label` here instead of `txt` to stay consistent with the other field, and put it at the front also